blogposts/structural-code-search-at-scale-from-your-browser.md

```diff
@@ -0,0 +1,305 @@
+---
+title: Structural code search. At scale and from your browser.
+author: Rijnard van Tonder
+authorUrl: https://twitter.com/rvtond
+publishDate: 2020-01-24T10:00-07:00
+tags: [blog]
+slug: structural-code-search-at-scale-from-your-browser
+heroImage: https://TODO
+published: true
+---
+
+We're introducing a richer way to search code at Sourcegraph with structural
+code search. And we're making it available at scale.
+
+## What _is_ structural code search?
+
+Structural code search is the idea that you can search for _syntactic
```

Comment:
Text formatting preference is to have each paragraph on a single line and use soft line wrapping in your editor but up to you.